drivers/barometer_bmp085.cpp

```diff
@@ -250,7 +249,7 @@ bool Barometer_BMP085::update(void)
             }
 
             // remove bias
-            altitude_gf_ = altitude_filtered - altitude_bias_gf_;
+            altitude_gf_ = altitude_filtered;
```

Comment:
done in https://github.com/lis-epfl/MAVRIC_Library/pull/311/commits/abad5bd90f89aa8de0fdcd1a8bfed8e493b2ec32